main.css

```diff
@@ -67,5 +67,5 @@ h1{
   background-color: rgb(100,100,100);
   padding: 0.5em;
   margin: 0.5em;
-  cursor: default;
+  cursor: pointer;
```

Comment:
But then how do people know they can click it?
Oh also it only clicks on label click right now, the rectangle isn't entirely clickable.